README.md

```diff
@@ -3,42 +3,335 @@
+
+The package relies on other packages as building blocks and parallelisation tools:
+
+* [ParallelStencil.jl](https://github.com/omlins/ParallelStencil.jl)
+* [ImplicitGlobalGrid.jl](https://github.com/omlins/ImplicitGlobalGrid.jl)
+* [GeoParams.jl](https://github.com/JuliaGeodynamics/GeoParams.jl)
+* [JustPIC.jl](https://github.com/JuliaGeodynamics/JustPIC.jl)
+
 
 The package serves several purposes:
 
-  * It reduces code duplication between several applications, e.g. [PseudoTransientStokes.jl](https://github.com/PTsolvers/PseudoTransientStokes.jl)
-  and [PseudoTransientAdjoint.jl](https://github.com/PTsolvers/PseudoTransientAdjoint.jl)
   * It provides a collection of solvers to be used in quickly developing new applications
   * It provides some standardization so that application codes can
 
-     - more easily "add more physics"
-     - more easily switch between a psuedo-transient solver and another solver (e.g. a direct solve or a multigrid method)
+     - more easily handle local material propoerties through the use of [GeoParams.jl]((https://github.com/JuliaGeodynamics/GeoParams.jl))
+     - more easily switch between a psuedo-transient solver and another solver (e.g. an explicit thermal solvers)
 
-  * It provides a natural place to describe performance benchmarks for the solver routines
-  * It defines useful solvers to be encapsulated and used from non-Julia applications
+  * It provides a place to describe performance benchmarks for the solver routines
   * It provides a natural location for contributions of new solvers for use by the larger community
 
 We include several miniapps, each designed to solve a well-specified benchmark problem, in order to provide
 
   - examples of high-performance usage,
   - bases on which to build more full-featured application codes
   - cases for reference and performance tests
-  - JustRelax.jl's entries in "bake offs"
-  - tests and examples of interfaces with other packages applications might use, in particular
-    - CompGrids.jl
-    - PETSc.jl
 
-JustRelax.jl is used in the following applications:
 
-  * TODO link to all applications using the package here (crucial for early development)
+## Installation
+
+`JustRelax.jl` is not yet registered (we are in the process), however it can be installed by cloning the repository. If you have navigated the terminal to the directory you cloned `JustRelax.jl`, you can test the package by running the following commands:
+
+```julia
+using JustRelax
+julia> ] 
+  pkg> activate .
+  pkg> instantiate
+  pkg> test JustRelax
+```
+The test will take a while, so grab a coffee or tea. 
+
+## Usage
+
+For the advection of physical fields, this package relies on the Particles-In-Cell method implemented in [JustPIC.jl](https://github.com/JuliaGeodynamics/JustPIC.jl). Depending on where you want to run `JustRelax.jl` (CPU or GPU) you will need to specify the Backend. Run this command in the REPL and restart Julia:
```

Comment:
```suggestion
JustRelax can execute on different backends, currently CPUs, and Nvidia and AMD GPUs. Execute the following command in the REPL to select the backend, and restart Julia:
```

Reply:
Following "For the advection of physical fields, this package relies on the Particles-In-Cell method implemented in [JustPIC.jl](https://github.com/JuliaGeodynamics/JustPIC.jl). " I would put somewhere else.